src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,48 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_STORAGE_SIZE = 10;
+    private final K[] keys;
+    private final V[] values;
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
size is 0 by default